Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Task.java

```diff
@@ -345,6 +350,24 @@ public void setProcess(Process process) {
         this.process = process;
     }
 
+    /**
+     * Get project(s). If the task belongs to a template, the projects are in
+     * the template. If the task belongs to a process, the projects are in the
+     * process.
+     *
+     * @return value of project(s)
+     */
+    @Transient
+    public List<Project> getProjects() {
+        if (Objects.nonNull(process)) {
+            return Arrays.asList(process.getProject());
+        } else if (Objects.nonNull(template)) {
+            return template.getProjects();
+        } else {
+            return Collections.emptyList();
+        }
+    }
+
```

Comment:
This is something which should be added to the TaskService. Please don't add functionality to beans.